resources.html

```diff
@@ -57,7 +57,7 @@
 							<a class="page-scroll" href="./index.html">Home</a>
 						</li>
                         <li>
-                            <a class="page-scroll" href="http://54.191.129.39/">Get Started</a>
+                            <a class="page-scroll" href="http://54.191.129.39/" target="_blank">Get Started</a>
```

Comment:
yeah, you're right it was. since we're messing with the line anyway, might as well fix it.